tools/cldr-apps/js/src/views/InfoPanel.vue

```diff
@@ -20,6 +20,7 @@
         </a-button>
         <span class="i-am-info-panel">Info Panel</span>
         <a-button
+          v-if="($locale.value) && ($xpath.value)"
```

Comment:
this could be implemented without globals, simply by calling a function here that in turn calls getCurrentLocale and getCurrentId

for example, see the function canBeHidden in DashboardWidget.vue

Reply:
It's actually different.  $locale here is reevaluated dynamically when the locale changes.   Calling getCurrentLocale() would not have the same behavior.

Reply:
details: https://vuejs.org/guide/essentials/reactivity-fundamentals.html

Reply:
reactivity is great; reactive global variables, not so great, and not necessary

the info panel gets regenerated anyway whenever the locale or path changes, or at least it always has been that way until now

the persistence of a component in the info panel when the locale or path changes is an interesting idea and I'm not opposed to it, if it's implemented without global variables

getCurrentLocale and getCurrentId will work -- at most, a few lines of code might be needed to make the component refresh when the info panel is regenerated -- I'd be happy to implement it

another alternative is for cldrInfo to call setters for locale/path in the component when the info panel is regenerated -- that way locale/path can be reactive (and not global) -- theoretically that could prevent some flicker in the UI, though realistically it's unlikely to be a perceptible difference given that the rest of the window is also updating to show different data for the new locale/path -- again, I'd be happy to implement it